projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -463,4 +468,47 @@ public function wp_ajax_jitm_dismiss() {
 		}
 		wp_die();
 	}
+
+	/**
+	 * Adds a notice above each settings page while using the Classic view to indicate
+	 * that the Default view offers more features. Links to the default view.
+	 *
+	 * @return void
+	 */
+	public function add_settings_page_notice() {
+		if ( ! is_admin() ) {
+			return;
+		}
+
+		// Show the notice for the following screens and map them to the Calypso page.
+		$screen_map = array(
+			'options-general'    => 'general',
+			'options-writing'    => 'writing',
+			'options-reading'    => 'reading',
+			'options-discussion' => 'discussion',
+		);
+
+		$mapped_screen = isset( $screen_map[ get_current_screen()->id ] )
```

Comment:
`get_current_screen()` can return `null`: https://developer.wordpress.org/reference/functions/get_current_screen/#return

Need to make sure we have a valid screen object too.

Reply:
I added a check to ensure it is a WP_Screen